crates/solidity/inputs/language/src/lib.rs

```diff
@@ -5,6 +5,9 @@
 //! 2. Errors are reported only once, instead of repeating for every crate.
 //!
 //! Call the [`SolidityLanguageExtensions::load_solidity`] method to load the precompiled language definition.
+pub mod dsl;
```

Comment:
I suggest either making `dsl` private, or removing the `use dsl::*`, so that other crates have only one path to import the types in there, and make it easier to refactor/search for imports.

Reply:
done